pkg/eve/runme.sh

```diff
@@ -111,6 +111,10 @@ do_version() {
   echo /bits/*.squash | sed -e 's#/bits/rootfs-##' -e 's#.squash##' >&3
 }
 
+do_build_config() {
+  cat /bits/linuxkit-eve-config.yaml
```

Comment:
Can we use unsquashfs to etract the file from rootfs itself instead of storing another file?

Reply:
It is a few hundred bytes at most, and we already have lots of duplication in here. Also, `unsquashfs` doesn't like extracting to stdout, and so you have to set a directory, extract there, `cat` it, and then delete it. And if the dir already exists, `unsquashfs` fails with error. I sort of would prefer to leave it as is.

Reply:
I mean something like `unsquashfs -cat /bits/rootfs.img etc/linuxkit-eve-config.yml >&3`. And no additional logic required in Makefile.

Reply:
I'd be ok with that. I'll try it soon.

Reply:
I'd be very much for the same approach @giggsoff is advocating -- personally I hate maintaining consistency of same artifact in different places.

Also, this eve OCI image has always been meant to be a tool for manipulating what's inside of rootfs anyway

Reply:
Yeah, that actually works. I am ok with that. I will update.

Reply:
Works like a charm. Good call @giggsoff